portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,100 @@
+:root {
+    --target-color: red; 
+}
+
+body {
```

Comment:
Make body take up the entire space of the page, right now if you go outside the body you get a pointer with a circle behind it which is much less cool than you intended.

You can use the Developer Tools in Chrome to find the <body> tag in the Elements section, set the background-color attribute to blue, and play with the settings until it looks like you want.  That way you can see clearly how much space it is taking up (also test how it works when you resize the screen).